coordinator/cql/src/main/java/org/apache/cassandra/stargate/metrics/ClientMetrics.java

```diff
@@ -290,7 +290,12 @@ public ConnectionMetricsImpl(ClientInfo clientInfo) {
               .map(clientInfoTagProvider::getClientInfoTags)
               .orElse(Tags.empty());
 
-      requestsProcessed = meterRegistry.counter(REQUESTS_PROCESSED_METRIC, tags);
+      Tags tagsByDriver =
```

Comment:
why is tags and tags by driver separated now? Is it just to keep in in a single metric?

Reply:
This was at my request in order to cause the least disruption and potential metrics expansion. I'm not sure we ever look at the other metrics.